### DISCUSSION TITLE
How do I use the screenmirroring feature ?

### DISCUSSION THREAD

**Discussion by Vasmares** [BimmerGestalt/AAIdrive#426] [Q&A] [answered]
Hi,

my MINI F54 ID5 is successfully connnected with the app and I can use all the supported Android App.
The AddonBranch says, that screen mirroring is already implemented:
https://github.com/BimmerGestalt/IDriveConnectAddons

Did I miss something ?

**Comment by hufman** [comment, marked as answer]
Good sleuthing! The IDriveConnectAddons repository is a collection of additional apps that you can install to add extra features, and are not provided with AAIdrive. Besides being unrelated to the core premise of implementing Android Auto for IDrive, they are primarily small apps as examples for other people to learn from and develop their own apps. You can find more discussion [over here](https://github.com/BimmerGestalt/AAIdrive/discussions/421).
